Index: src/.env
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/.env b/src/.env
deleted file mode 100644
--- a/src/.env	(revision 952a24e18360e09d0d15b23ebc5fdd8571bf10dc)
+++ /dev/null	(revision 952a24e18360e09d0d15b23ebc5fdd8571bf10dc)
@@ -1,1 +0,0 @@
-REACT_APP_OPENAI_API_KEY='sk-my-service-account-za18K93YHebAlcU0liKKT3BlbkFJITPRdFUkfCxxpwM5txNz'
\ No newline at end of file
